.github/workflows/ci.yml

```diff
@@ -24,64 +24,61 @@ jobs:
     - uses: actions/checkout@v4
       with:
         submodules: recursive
-    - run: scripts/toolchain-before.sh risc0
     - uses: actions-rs/toolchain@v1
       with:
-        profile: minimal
-    - run: scripts/toolchain-after.sh risc0
-    - uses: risc0/risc0/.github/actions/sccache@release-0.19
+          toolchain: stable
+          profile: minimal
     - name: Install cargo-binstall
       uses: cargo-bins/cargo-binstall@v1.6.4
-    - name: Install cargo-risczero
-      run: |
-        cargo binstall -y --force cargo-risczero
-        cargo risczero install
-    - name: Build risc0 guest target
-      run: cargo run --bin risc0-builder
+    - uses: risc0/risc0/.github/actions/sccache@release-0.19
+    - name: Install risc0
+      run: TARGET="risc0" CI=1 make install
     - name: Build risc0 prover
-      run: cargo build --release --features risc0
+      run: make risc0
     - name: Test risc0 prover
-      run: cargo test -p risc0-driver --features enable
+      run: make test_risc0
 
   build-test-sp1:
     name: Build and test sp1
     runs-on: ubuntu-latest
     timeout-minutes: 60
     steps:
-      - uses: actions/checkout@v4
-      - run: scripts/toolchain-before.sh sp1
-      - uses: actions-rs/toolchain@v1
-        with:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+    - uses: actions-rs/toolchain@v1
+      with:
+          toolchain: nightly-2024-02-06
```

Comment:
A bit unfortunate that the toolchain version also needs to be specified here for it to download the required rust version. But I don't know another way right now.

Reply:
But that requires copying stuff again which I guess we want to avoid. A bit strange as well that the toolchain file makes it download the right rust version but specifying the version directly on the command line doesn't.